milton-manager/src/main/resources/application.conf

```diff
@@ -4,5 +4,7 @@ ktor {
     }
 }
 milton {
-    url = "https://us-central1-dsouza-proving-ground.cloudfunctions.net/scribe"
+    processorHost = "https://us-central1-dsouza-proving-ground.cloudfunctions.net/scribe"
+    projectName = "milton-273820"
+    googleAuthClientId = "1044906185219-4vkbn44qlm9kte30e1teg8v0fjks11c0.apps.googleusercontent.com"
```

Comment:
Don't forget our conversation about disabling the testing localhost integration.